### DISCUSSION TITLE
Allow dynamic values in advanced filters

### DISCUSSION THREAD

**Discussion by sidartaveloso** [directus/directus#6427] [Feature Requests]
With #6413 merged, user can't type commands on date filter, become impossible enter dynamic values like $NOW.

![image](https://user-images.githubusercontent.com/144222/122846284-e49b8400-d2db-11eb-938a-2d23aff710a2.png)

Should the Datetime interface allow the user to enter the type value by typing? 

---

### Project details
Directus Version: 9.0.0-rc.78
Environment: production
OS: Linux 5.4.0-66-generic
Node: 14.17.0

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
